src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -15,22 +15,24 @@ namespace Microsoft.NET.Sdk.WorkloadManifestReader
 {
```

Comment:
I think we should add a nullable enable directive here?

Reply:
I don't think we need it specifically in the file, I think it's set at the project level.  This file is compiled into 4 different projects, and I got errors when I didn't have the nullability declarations correct, so it's already enabled for at least some of the projects compiling it.